### ISSUE TITLE
Errors in trickle file runtimes aren't identified by line

### ISSUE THREAD

**Issue by nickjmh** [tremor-rs/tremor-runtime#1111]
**Problem**
Errors within trickle files are not identified beyond the filename during tremor_runtime startup:
```
2021-07-07T12:59:56.575560503-04:00 ERROR tremor::server - error: An error occurred while loading the file `/etc/tremor/config/main.trickle`: failed to load trickle script: /etc/tremor/config/main.trickle
```

**Notes**

Found in 0.11.4

**Comment by mfelsche** [comment]
Thanks @nickjmh for reporting this!

I tried to reproduce this, and we do emit a detailed hygienic error message, the problem though is that we print it to stderr instead of logging it. We should only use one method of notifying users about errors and that is the logging framework in use, so that when logging is setup to log to a file, users dont have to investigate the process stderr for additional information.

Here is an example output (it even has colors, that are lost in that snippet):

```
[2021-07-07T19:34:52Z INFO  tremor_runtime] Loading configuration from temp/scratch.trickle
Error: 
    3 | #  should be isolated from another
    4 | # 
    5 | select {"foo": event.old} meta {"class": } from in into out;
      |                           ^^^^ Found the token `meta` but expected one of `!=`, `#!config`, `%`, `&`, `)`, `*`, `+`, `,`, `-`, `/`, `:`, `;`, `<`, `<<`, `<=`, `<end-of-stream>`, `==`, `=>`, `>`, `>=`, `>>`, `>>>`, `]`, `^`, `and`, `case`, `create`, `default`, `define`, `end`, `from`, `group`, `into`, `mod`, `of`, `or`, `script`, `select`, `when`, `xor`, `}`
    6 | select {"foo": event.old} from in into out;
    7 | 

[2021-07-07T19:34:52Z ERROR tremor::server] error: An error occurred while loading the file `temp/scratch.trickle`: failed to load trickle script: temp/scratch.trickle
[2021-07-07T19:34:52Z ERROR tremor::server] We are SHUTTING DOWN due to errors during initialization!
```

The lines following `Error:` should be part of a log message.

**Comment by Licenser** [comment]
I'm closing this as it will be solved by #1120 which is about resolving the cause of this issue.
